### DISCUSSION TITLE
No response from Ikea STYRBAR E2001/E2002

### DISCUSSION THREAD

**Discussion by nimo-7** [Koenkk/zigbee2mqtt#16855] [Devices]
I have home assistant running as a virtual machine, with mosquitto and zigbee2mqtt. 

The problem is that when I pair the Ikea STYRBAR E2002 it is found, interviewed and paired, but cannot control anything. I tried to set up node red and create a debug setup where I have a mqtt in going into a debugger, but no matter what I press nothing is output to the debug window. I used a tutorial from YouTube on how to do this. 
I also have a couple of lightbulbs that I paired (to be controlled by the remote) and they work fine.

As far as I can see, the STYRBAR should be compatible with zigbee2mqtt. I have unpaired and paired many times trying out stuff I found in the net, but to no avail.

Does anyone how an idea on what to do?

**Comment by jarekj9** [comment]
Did you find solution ?
I'm running my home assistant for years and have few zigbee devices (also older ikea remote). I just bought the STYRBAR remote because its on the list of supported devices.
In docker logs on zigbee2mqtt i see 
`Received message from unsupported device with Zigbee model 'undefined' and manufacturer name 'undefined'`

I though update would solve the issue so i pulled new image and recreated container, but it did not change anything. I have latest image version created on 1st May. I use conbee 2.

**Reply by Astrofreak85** [reply]
If you are using ConBee, there seems to be a bug in newer FW...had the same issue.
Try flashing 0x26580700, this works fine for me

**Comment by jarekj9** [comment]
Nevermind, solved it.
I removed device from database.db and configuration.yaml and once again recreated container. Then it paired succesfully.
I think the problem was, that i paired it first on the old version, where it was unsupported.

**Comment by Busta999** [comment]
With the latest update to zigbee2MQTT as of 7 May and with a device fw of 1.0.024 I had no issues connecting and operating this really nice, solid switch, so much looking at replacing tuya 4 switches with them.
